composer.json

```diff
@@ -41,16 +41,15 @@
         "knplabs/gaufrette": "^0.1.6 || ^0.2",
         "imagine/imagine": "^0.6",
         "kriswallsmith/buzz": "^0.15",
-        "jms/serializer-bundle": "^0.13 || ^1.0",
-        "sonata-project/intl-bundle": "^2.2"
+        "jms/serializer-bundle": "^0.13 || ^1.0"
     },
     "require-dev": {
         "sonata-project/doctrine-orm-admin-bundle": "^3.0",
         "sonata-project/admin-bundle": "^3.1",
         "sonata-project/formatter-bundle": "^3.0",
         "sonata-project/datagrid-bundle": "^2.2",
         "sonata-project/block-bundle": "^3.1.1",
-        "sonata-project/seo-bundle": "^2.0",
+        "sonata-project/seo-bundle": "^2.1",
```

Comment:
Because seo-bundle 2.0 is a very old release which contains a TestBase no longer included on 2.1, that's why the build broke but not when using lower deps.